component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java

```diff
@@ -186,10 +188,14 @@ public Response getGroupsTree(@Context UriInfo uriInfo,
       }
   )
   public Response createGroup(GroupImpl group) throws Exception {
+    String regex = "^[a-zA-Z0-9-_]+$";
+    Pattern pattern = Pattern.compile(regex);
+    Matcher matcher = pattern.matcher(group.getGroupName());
+    boolean isValid =  matcher.matches();
     if (group == null) {
       return Response.status(Response.Status.BAD_REQUEST).entity("Group object is required").build();
     }
-    if (StringUtils.isBlank(group.getGroupName())) {
+    if (StringUtils.isBlank(group.getGroupName()) || !isValid) {
```

Comment:
Could you create a specific error for invalid group name instead of using the Required option.

Reply:
done